scripts/common_test.py

```diff
@@ -347,3 +348,30 @@ def _mock_subprocess_call(unused_command):
 
         with self.swap(subprocess, 'call', _mock_subprocess_call):
             common.install_npm_library('library_name', 'version', 'path')
+
+    def test_ask_user_to_confirm(self):
+        def mock_input():
+            return 'Y'
+        with self.swap(python_utils, 'INPUT', mock_input):
+            common.ask_user_to_confirm('Testing')
+
+    def test_get_personal_access_token_with_valid_token(self):
+        # pylint: disable=unused-argument
```

Comment:
If you call the unused argument unused_prompt (i.e. add an "unused_" prefix), I think you don't need this pragma.

Ditto below.

Reply:
Oh yes, I have used that prefix where I can. I cannot use it here since `prompt` is being passed as a named argument and if we use the `unused_` prefix here, we will get an error that the function got an unexpected argument `prompt`.